gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -279,8 +279,8 @@ gst_tensor_time_sync_buffer_from_collectpad (GstCollectPads * collect,
   GstTensorsConfig in_configs;
   GstClockTime base_time = 0;
   guint i, n_mem;
-  GstMemory *in_mem[NNS_TENSOR_SIZE_LIMIT];
-  tensor_format in_formats[NNS_TENSOR_SIZE_LIMIT];
+  GstMemory *in_mem[NNS_TENSOR_SIZE_LIMIT + NNS_TENSOR_SIZE_EXTRA_LIMIT];
```

Comment:
I wonder if one define to replace 'NNS_TENSOR_SIZE_LIMIT + NNS_TENSOR_SIZE_EXTRA_LIMIT' cannot be used.

Reply:
Your suggestion is valid, I'll define the macro in future patches. Thanks.